.github/ISSUE_TEMPLATE/improvement.yml

```diff
@@ -1,5 +1,6 @@
 name: "🚀 Improvement Request"
 description: Suggest an improvement to the product.
+type: "feature"
```

Comment:
Is the type correct?

Reply:
@LakshanWeerasinghe yes we have a known limitation which is already mentioned in the PR description